### DISCUSSION TITLE
Ender Extender with 400x400 bed & CRTouch- how are UBL maximum probable point determined??

### DISCUSSION THREAD

**Discussion by dohlin** [Jyers/Marlin#1735] [Q&A]
Quick question that I'm struggling to find the answer to.  I've recently upgraded my E3 v2 to an Ender extender 400x400 kit, and made the corresponding changes to x and y bed sizes in the firmware.

I've also messed with various different UBL grid sizes too, and am currently trialing a basic 5x5 grid.  My problem is that when I go to create a mesh, it only hits 9 of the points (the center most 9).  I understand that ABL doesn't hit all points all the time, but what I don't understand how the number of 'probable points' is determined and if there's anything else I can tweak.  Right now there's a solid ~3+" on all four sides of the 9 points the probe measures...so it certainly seems like that is a LOT of room that should be reachable by more probe points.

Prior to upgrade, my stock E3 v2 would hit all 9 points on a 3x3 grid sizes, so is there anything I can tweak to get more or all of the 5x5 grid size?  Thanks!!

**Comment by justice4no1** [comment]
I’ve the same issues with my CR touch and I was told the CR touch struggles with anything over 9 touches and to get a BL touch or 3D Touch “BL touch clone” mine has been ordered

**Reply by dohlin** [reply]
If that's seriously the best option that really sucks.  I previously got used to just manual leveling and to be honest once it was in place it was less of a pain that a bed sensor.  Could always just go back to that I suppose.  Anyone else have any other advice here?  Many thanks!
